src/Util/filter-utilities.js

```diff
@@ -0,0 +1,75 @@
+import { initialState } from '../Reducers/filter-reducer'
+
+const useFilterFunc = (filterFuncArr, data, state) => {
+    let copiedData = [...data]
+    for (let filterFunc of filterFuncArr) {
+        copiedData = filterFunc(copiedData, state)
+    }
+    return copiedData;
+}
+
+const sortItems = (data, state) => {
+    const sortedData = [...data]
+    if (state.sortBy === 'RATING') {
+        sortedData.sort((item1, item2) =>
+            item2.rating - item1.rating
+        )
+    } else if (state.sortBy == 'LOW_TO_HIGH') {
+        sortedData.sort((item1, item2) =>
+            parseInt(item1.currentprice) - parseInt(item2.currentprice)
+        )
+    } else if (state.sortBy == 'HIGH_TO_LOW') {
+        sortedData.sort((item1, item2) =>
+            parseInt(item2.currentprice) - parseInt(item1.currentprice)
```

Comment:
I am not sure about the need for parseInt, if you store price as Number type, you won't have to write parseInt everytime you use it.

Reply:
i was just trying to be on the safer side but will remove it